README.md

```diff
@@ -0,0 +1,13 @@
+# Gmp-detector
+
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
```

Comment:
The README says that the aim is to find "all" the GMP functions, however, the script you made is not a general-purpose function. It's okay if the repository has that nice aim, but can you make some additional description to make it clear that the current script can not detect all the GMP functions?

Reply:
Of course, thank you for your remark.